app/models/businessmatching/BusinessMatching.scala

```diff
@@ -126,13 +126,14 @@ case class BusinessMatching(
     activities map { a =>
       a.businessActivities.map {
         case AccountancyServices => Messages("businessactivities.registerservices.servicename.lbl.01")
-        case BillPaymentServices => Messages("businessactivities.registerservices.servicename.lbl.02")
-        case EstateAgentBusinessService if estateAgent => Messages("businessactivities.registerservices.servicename.lbl.03.agent")
-        case EstateAgentBusinessService => Messages("businessactivities.registerservices.servicename.lbl.03")
-        case HighValueDealing => Messages("businessactivities.registerservices.servicename.lbl.04")
-        case MoneyServiceBusiness => Messages("businessactivities.registerservices.servicename.lbl.05")
-        case TrustAndCompanyServices => Messages("businessactivities.registerservices.servicename.lbl.06")
-        case TelephonePaymentService => Messages("businessactivities.registerservices.servicename.lbl.07")
+        case ArtMarketParticipant => Messages("businessactivities.registerservices.servicename.lbl.03")
+        case BillPaymentServices => Messages("businessactivities.registerservices.servicename.lbl.03")
```

Comment:
Label with the numeric value 03 appears to be used for two different activities here.

Reply:
Fixed